RemoteWiki.php

```diff
@@ -3,6 +3,7 @@ class RemoteWiki {
 	protected $db;
 
 	private function __construct( $dbname, $sitename, $language, $private, $closed, $closedDate, $inactive, $inactiveDate, $settings ) {
+		$this->db = wfGetDB( DB_MASTER, [], 'metawiki' );
```

Comment:
Remove this

Reply:
And change line 47 to "$res = wfGetDB( DB_MASTER, [], 'metawiki' )->selectField("